src/eko/output/struct.py

```diff
@@ -650,42 +603,61 @@ def bootstrap(dirpath: os.PathLike, theory: dict, operator: dict):
         (dirpath / PARTSDIR).mkdir()
         (dirpath / OPERATORSDIR).mkdir()
 
-    @staticmethod
-    def extract(path: os.PathLike, filename: str) -> str:
-        """Extract file from disk assets.
-
-        Note
-        ----
-        At the moment, it only support text files (since it is returning the
-        content as a string)
+    @classmethod
+    def open_tar(cls, path: os.PathLike):
```

Comment:
Since tar is the only file format supported, I propose renaming the function:
```suggestion
    def load(cls, path: os.PathLike):
```